release/models/gnsi/openconfig-gnsi-certz.yang

```diff
@@ -30,12 +30,18 @@ module openconfig-gnsi-certz {
     "This module provides a data model for the metadata of gRPC credentials
     installed on a networking device.";
 
-  oc-ext:openconfig-version "0.6.0";
+  oc-ext:openconfig-version "0.7.0";
+
+  revision 2024-03-14 {
+    description
+      "remove ssl-profile-id comment";
+    reference "0.7.0";
+  }
 
   revision 2024-03-05 {
     description
       "rename access/reject counters";
-    reference "0.6.0";
+    reference "0.7.0";
```

Comment:
one of the linters from gcbrun will complain about this if i leave it as-is. 
it will say that the file was modified without updating the reference version, which is why i updated this